src/main/java/net/praqma/tracey/cli/Tracey.java

```diff
@@ -38,6 +34,7 @@ public static void main(String[] args) throws Exception {
         sayParser.addArgument("-n", "--node").dest("node").help("The URL of the RabbitMQ server");
```

Comment:
Do we need that? I mean, The broker and the routing info will use default values in any way if no new data provides